packages/lesswrong/server/callbacks/userCallbacks.ts

```diff
@@ -229,9 +229,6 @@ getCollectionHooks("Users").editAsync.add(async function subscribeToForumDigest
         latitude,
         longitude,
       }}),
-      merge_fields: {
-        FNAME: newUser.fullName
-      },
```

Comment:
the fullName field is hidden on the EA Forum. At first I leaned toward replacing this with displayName, but on reflection my sense is that "Dear username123" was a worse greeting than a generic one. (Related, Mailchimp has a templating system that allows for greetings to be conditional on the existence of fields like FNAME.) I'm open to using displayName here if you think that's preferable.